recipes/cppserver/all/conanfile.py

```diff
@@ -0,0 +1,70 @@
+from conans import ConanFile, CMake, tools
+import os
+import glob
+
+
+class CppServer(ConanFile):
+    name = "cppserver"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/chronoxor/CppServer"
+    description = "Ultra fast and low latency asynchronous socket server and" \
+        " client C++ library with support TCP, SSL, UDP, HTTP, HTTPS, WebSocket" \
+        " protocols and 10K connections problem solution."
+    topics = ("network", "socket", "async", "low-latency")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"fPIC": [True, False],
+               "shared": [True, False]}
+    default_options = {"fPIC": True,
+                       "shared": False}
+    requires = ["asio/1.17.0", "openssl/1.1.1i", "cppcommon/cci.20201104"]
+    generators = "cmake", "cmake_find_package"
+    exports_sources = ["patches/**", "CMakeLists.txt"]
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def _configure_cmake(self):
+        if not self._cmake:
+            self._cmake = CMake(self)
+            self._cmake.definitions["CPPSERVER_MODULE"] = "OFF"
+            self._cmake.configure(build_folder=self._build_subfolder)
+        return self._cmake
+
```

Comment:
```suggestion

    @property
    def _compilers_minimum_version(self):
        return {
            "gcc": "9",
            "Visual Studio": "15",
            "clang": "5",
            "apple-clang": "10",
        }

    def configure(self):
        if self.settings.compiler.get_safe("cppstd"):
            tools.check_min_cppstd(self, "17")
        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
        if not minimum_version:
            self.output.warn("cppserver requires C++17. Your compiler is unknown. Assuming it supports C++17.")
        elif tools.Version(self.settings.compiler.version) < minimum_version:
            raise ConanInvalidConfiguration("cppserver requires a compiler that supports at least C++17")

```

According to 5d837e3c255ff7e2e03af9e83ed0d74e3da3bc21396deb683e40a647ca8301ecR87, it requires C++17